test/e2e/tests/errors.spec.js

```diff
@@ -737,3 +744,92 @@ describe('Sentry errors', function () {
     });
   });
 });
+
+describe.only('Sentry App State Mask', function () {
```

Comment:
I will extract this to a separate file, and put the `matchesSnapshot` function in a shared file.

Reply:
remove the `.only`